pages/contact.js

```diff
@@ -1,7 +1,101 @@
-import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEnvelope,faPhoneFlip,faAngleDown } from '@fortawesome/free-solid-svg-icons'
+import Footer from '../components/footer/Footer';
```

Comment:
remove footer, it's already included in layout component so no need to import separately